client/my-sites/sites/index.jsx

```diff
@@ -93,6 +93,9 @@ class Sites extends Component {
 			case 'home':
 				path = translate( 'My Home' );
 				break;
+			case 'hosting-admin':
+				path = translate( 'SFTP & MySQL' );
```

Comment:
http://calypso.localhost:3000/hosting-admin if folks visit directly

<img width="374" alt="Screen Shot 2019-11-04 at 1 14 14 PM" src="https://user-images.githubusercontent.com/1270189/68158704-09739580-ff05-11e9-998a-70cf3c3c6dfd.png">

Not a blocker, but let's align the page title with what we pick for copy in the section.

<img width="241" alt="Screen Shot 2019-11-04 at 1 15 14 PM" src="https://user-images.githubusercontent.com/1270189/68158766-2b6d1800-ff05-11e9-8573-8e7aff3f757d.png">

Reply:
https://github.com/Automattic/wp-calypso/pull/37321